libs/modelviewer/src/components/tags-overlay/TagsOverlay.tsx

```diff
@@ -25,58 +25,56 @@ export const TagsOverlay = (): JSX.Element => {
     setSelected(tag);
   };
 
-  // useEffect(() => {
-  //   console.log(filterTags);
-  // }, [filterTags]);
-
   return (
     <div>
-      {overlayTags.map((tag, index) => {
-        const isSelected = selected === tag.tagNo;
-        return (
-          <div
-            key={`${tag.tagNo}_${index}`}
-            title={titleResolver ? titleResolver(tag) : tag.tagNo}
-            onClick={() => {
-              echoInstance?.viewer.cameraManager.fitCameraToBoundingBox(
-                tag.boundingBox,
-                defaultRadiusFactor
-              );
-            }}
-          >
-            <RevealHtmlOverlayWrapper
-              overlayTool={overlayTool.current}
-              position3d={tag.position}
-              tagNo={tag.tagNo}
-              aabb={tag.aabb}
+      {overlayTags
+        .filter((ot) => filterTags.includes(ot.tagNo))
+        .map((tag, index) => {
+          const isSelected = selected === tag.tagNo;
+          return (
+            <div
+              key={`${tag.tagNo}_${index}`}
+              title={titleResolver ? titleResolver(tag) : tag.tagNo}
+              onClick={() => {
+                echoInstance?.viewer.cameraManager.fitCameraToBoundingBox(
+                  tag.boundingBox,
+                  defaultRadiusFactor
+                );
+              }}
             >
-              {CustomOverlayComponent ? (
-                <div
-                  onClick={(e) => {
-                    e.stopPropagation();
-                    setSelected(tag.tagNo);
-                  }}
-                >
-                  <CustomOverlayComponent
-                    {...tag}
-                    index={index}
-                    clearSelection={() => setSelected(tag.tagNo)}
+              <RevealHtmlOverlayWrapper
+                overlayTool={overlayTool.current}
+                position3d={tag.position}
+                tagNo={tag.tagNo}
+                aabb={tag.aabb}
```

Comment:
Can we find a better name for aabb?

Reply:
This is a name that comes from the @equinor/echo-3d-viewer package.
I think we should follow the same naming conventions as they use in their models to avoid confusion.

Reply:
After a discussion, we keep the aabb name